src/apps/Console/Console.js

```diff
@@ -152,7 +152,9 @@ function Prompt({ command, handleChange, handleSubmit, loading }) {
       parsedCommand[0] === 'install' && parsedCommand.length === 4
     const isValidExec =
       parsedCommand[0] === 'exec' && parsedCommand.length === 3
-    const isValidAct = parsedCommand[0] === 'act' && parsedCommand.length === 4
+    const isValidAct =
+      parsedCommand[0] === 'act' &&
+      (parsedCommand.length === 4 || parsedCommand.length === 5)
```

Comment:
After 3 months, I think we can totally do better sanitation of the commands 😆but let's do it in another PR!